docs/Configuration/Configuration.md

```diff
@@ -0,0 +1,149 @@
+﻿# KMD Logic Document Generation Configuration
+
+To configure a Logic Document Generation environment, you need to access [Logic Console](https://console.kmdlogic.io/) to do so.
+
+## Accessing Logic Document Generation Configuration module
+
+To access the Logic Document Generation Configuration module you will need to click on the menu labeled as `Document Generation` on the sidebar menu (highlighted in the image below).  
+
+![Image of Logic Console sidebar menu](./assets/image001.jpg)
+
+After doing that you will be presented with the `Logic Document Generation Configuration` module.
+
+![Image of Logic Console Document Generation Configuration module](./assets/image002.jpg)
+
+
+## Creating a Configuration environment
+
+To create a Document Generation Configuration click on the `ADD CONFIGURATION` button at the top right corner of the screen.
+
+You will be then presented with the following screen:
+
+![Image of Logic Console Create Document Generation Configuration screen](./assets/image003.jpg)
+
+The screen is divided in two sections, the `Configuration Settings` and the `Master Storage Settings`, we will explain them below in detail.
+
+### Configuration Settings
+
+![Image of Logic Console Create Document Generation Configuration - Settings](./assets/image004.jpg)
+
+- Configuration name:  
+_(Required)_ The configuration name, any non-empty string is valid.
+
+- ASPOSE license switch:  
+_(Required)_ In order to use `Logic Document Generation` your organization needs to have a valid ASPOSE license. Toggle this switch on to acknowledge that.
+
+- Levels:
+_(Optional)_ `Logic Document Generation` allows you to consume templates stored in multiple storages, it uses a hierarchical organization where each _**level**_ down the hierarchy is able to override a template found on its parent _**level**_.  
+By default this screen is pre-populated with two levels (Customer and Department), feel free to modify this to suit your needs.  
+A configuration environment can have zero, one or multiple levels.  
```

Comment:
I'd probably drop the zero (even though it's true). what's the point without a master level?

Reply:
If its zero, one or multiple levels, we're really only ruling out negatives :)
So we could maybe even lose the whole line.

Reply:
Not having levels means you can only have a Master entry and Master cannot have children entries.
Say you have only 1 level, that means Master can have children entries.
So zero is still a valid number of child levels.

Reply:
@danielhazel I agree, removing the paragraph may be better.

Reply:
Fixed in 100845b4e1e48294a117fb0efdc084d7ba920895